package.json

```diff
@@ -1,16 +1,13 @@
 {
-  "name": "react-from-markup-workspace",
+  "name": "react-from-html-workspace",
   "version": "0.0.0",
   "main": "index.js",
-  "author": "Simon Andrews <me@simonandrews.ca>",
+  "author": "Measured.co <hello@measured.co>",
```

Comment:
`Measured Co.` might be better, as standard way to write org name.

Reply:
Fixed, keep tripping up on this.